packages/state-transition/src/cache/epochCache.ts

```diff
@@ -692,6 +805,31 @@ export class EpochCache {
     };
   }
 
+  /**
+   * Return indices of validators who attestested in `attestation`
+   */
+  getAttestingIndices(fork: ForkSeq, attestation: Attestation): number[] {
```

Comment:
can this function use `this.isAfterElectra` instead of `fork`?

same with `getIndexedAttestation`